jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-client/src/main/java/org/jbpm/console/ng/es/client/editors/quicknewjob/QuickNewJobPopup.java

```diff
@@ -74,7 +74,7 @@
             UiBinder<Widget, QuickNewJobPopup> {
 
     }
-    private final Constants constants = Constants.INSTANCE;
+    private Constants constants = Constants.INSTANCE;
```

Comment:
hm, I think this could still be final, don't think that reassign is wanted for this variable.